web-client/src/app/components/action-item/action-item.component.stories.ts

```diff
@@ -0,0 +1,17 @@
+import { Meta, Story } from '@storybook/angular';
+import { ActionItemComponent } from './action-item.component';
+
+export default {
+  title: 'Components/ActionItemComponent',
+  component: ActionItemComponent,
+} as Meta;
+
+const Template: Story<ActionItemComponent> = (args: ActionItemComponent) => ({
+  props: args,
+});
+
+export const Default = Template.bind({});
+Default.args = {
+  icon: 'wallet',
+  title: 'My wallet',
```

Comment:
Minor thing. Capitalization in the designs are "My Wallet". It probably make sense to stay consistent with that in the stories.

Reply:
oh yea, in the stories it can be updated with the variables.
I just put that one quick for demo purposes.

Thanks for spotting!